@shared/api/internal.ts

```diff
@@ -605,18 +605,23 @@ export const getTokenPrices = async (tokens: string[]) => {
 };
 
 export const getDiscoverData = async () => {
-  const url = new URL(`${INDEXER_URL}/protocols`);
+  const url = new URL(
+    "https://freighter-backend-v2-stg.stellar.org/api/v1/protocols",
```

Comment:
hardcoded to staging URL. This will be changed to the prod URL before release

Reply:
@piyalbasu Why cant we use the ${INDEXER_BETA_URL} to construct only protocols URL here? Why cant we have more than 1 secret?

Reply:
Oh you're right - fixing this